turbopack/crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -1112,6 +1112,10 @@ impl FileSystemPath {
         self.fs().read(self).parse_json()
     }
 
+    pub fn read_json5(self: Vc<Self>) -> Vc<FileJsonContent> {
```

Comment:
nitpick: Do we need`#[turbo_tasks::function]` for this? (I'm not sure)

Reply:
`pub fn read_json` also didn't have it 🤷